### PR TITLE
use ATOL metrics to test log and sin tests

### LINKED ISSUE #114: CI failed on Mac platform

### PR DESCRIPTION
**Pull request by BruceDai**
Fixes #114. 
@fdwr @huningxin PTAL, thanks!

### CODE REVIEW

File: test/unary_test.js [hunk 1]
```diff
@@ -5,11 +5,15 @@ import * as unaryFunctions from '../src/unary.js';
 import * as utils from './utils.js';
 
 describe('test unary', function() {
-  function testUnary(op, input, expected, shape) {
+  function testUnary(op, input, expected, shape, atolTolerance) {
```
Comment (fdwr) [inline test/unary_test.js:8]: *atol* stands for *absolute tolerance*, and So *atolTolerance* would means *absolute tolerance tolerance*. How about either `absoluteTolerance` or just `atol`?

File: test/unary_test.js [hunk 2]
```diff
@@ -670,7 +674,7 @@ describe('test unary', function() {
           0.21339342411295945,
           -0.6558230571220807,
         ],
-        [3, 2, 2, 1]);
+        [3, 2, 2, 1], 1 / 1024);
```
Comment (fdwr) [inline test/unary_test.js:677]: (nit) Would feel cleaner/more consistent on a separate line like the other entities.

```suggestion
        [3, 2, 2, 1],
        1 / 1024);
```
Though, if a code reformatting tool is the cause of this, then sigh, never mind.

File: test/utils.js [hunk 3]
```diff
@@ -48,6 +48,15 @@ export function checkValue(tensor, expected, nulp = 0) {
   }
 }
 
+export function checkValueByATOL(tensor, expected, tolerance) {
```
Comment (fdwr) [inline test/utils.js:51]:
```suggestion
export function checkValueByATOL(actual, expected, tolerance) {
```
`    const actualValue = tensor.getValueByIndex(i);`

To be clearer, as `tensor` and `expected` are both *tensors*.

**Review by fdwr — APPROVED** [review]
Minor thoughts, else LGTM.

**Comment by BruceDai** [comment]
@fdwr I've updated commit to address your comments. Please take another review, thanks ! ☕

**Review by fdwr — APPROVED** [review]
👍

File: test/unary_test.js [hunk 4]
```diff
@@ -380,7 +384,8 @@ describe('test unary', function() {
           -1.0692689659512902,
           0.04121219038394666,
         ],
-        [3]);
+        [3],
+        1 / 1024);
```
Comment (huningxin) [inline test/unary_test.js:388]: Would it be clear to annotate the parameter? Do the same for the others.
```suggestion
        /*atol=*/1 / 1024);
```

Comment (BruceDai) [reply test/unary_test.js]: Done. 
Please take another look at  the new commit, thanks!

**Review by fdwr — APPROVED** [review]
👍

**Review by huningxin — APPROVED** [review]
LGTM

**Pull request merged**